src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -82,6 +82,29 @@ namespace picongpu
   } // end namespace: radFormFactor_CIC_1Dy
 
 
+  namespace radFormFactor_Gauss_spherical
+  {
+    class radFormFactor
```

Comment:
only a note: Types/classes must begin with capital letter 

Other classes are also written with small one therefore we can keep this.